templates/kubernetes/what-is-kubernetes.html

```diff
@@ -284,226 +305,339 @@ <h2 class="p-muted-heading u-align--center">
+            <div>
+              <button type="button" class="p-accordion__tab" id="tab17" aria-controls="tab17-section" aria-expanded="true">
+                Container runtime/ CRI
+              </button>
+            </div>
+            <section class="p-accordion__panel" id="tab17-section" aria-hidden="false" aria-labelledby="tab17">
+              <p>
+                The software responsible for running containers, by coordinating the use of system resources across containers.  Kubernetes can use different container runtimes (e.g. containerd, CRI-O) through the container runtime interface (CRI).              
+              </p>
+            </section>
           </li>
-          <li class="p-list__item is-crossed">Provide application-level services, such as middleware, databases and storage clusters out-of-the box. Such components can be integrated with k8s through add-ons.
+          <li class="p-accordion__group">
+            <div>
+              <button type="button" class="p-accordion__tab" id="tab18" aria-controls="tab18-section" aria-expanded="true">
+                CNI
+              </button>
+            </div>
+            <section class="p-accordion__panel" id="tab18-section" aria-hidden="false" aria-labelledby="tab18">
+              <p>
+                The Container Network Interface is a specification and a set of tools to define networking interfaces between network providers and Kubernetes.              
+              </p>
+            </section>
           </li>
-          <li class="p-list__item is-crossed">Provide nor dictate specific logging, monitoring and alerting components
+          <li class="p-accordion__group">
+            <div>
+              <button type="button" class="p-accordion__tab" id="tab19" aria-controls="tab19-section" aria-expanded="true">
+                CSI
+              </button>
+            </div>
+            <section class="p-accordion__panel" id="tab19-section" aria-hidden="false" aria-labelledby="tab19">
+              <p>
+                The Container Storage Interface is a specification for data storage tools and applications to integrate with Kubernetes clusters.              
+              </p>
+            </section>
+          </li>
+          <li class="p-accordion__group">
+            <div>
+              <button type="button" class="p-accordion__tab" id="tab20" aria-controls="tab20-section" aria-expanded="true">
+                Kubectl
+              </button>
+            </div>
+            <section class="p-accordion__panel" id="tab20-section" aria-hidden="false" aria-labelledby="tab20">
+              <p>
+                A command-line tool for controlling Kubernetes clusters.              
+              </p>
+            </section>
           </li>
         </ul>
+      </aside>
       </div>
     </div>
   </section>
 
-  <section class="p-strip--light is-deep">
+  <section class="p-strip is-deep">
     <div class="row">
       <div class="col-8">
         <h2>
```

Comment:
Do we need to add space between the title and the content? 

```suggestion
        <h2 class="u-sv3">
```